site/layouts/partials/scripts.html

```diff
@@ -26,36 +26,42 @@
   document.querySelectorAll('.btn-edit').forEach(btn => {
     btn.addEventListener('click', event => {
       const htmlSnippet = event.target.closest('.bd-code-snippet').querySelector('.bd-example').innerHTML
-      StackBlitzSDK.openBootstrapSnippet(htmlSnippet)
+
+      // Get extra classes for this example except '.bd-example'
+      const classes = Array.from(event.target.closest('.bd-code-snippet').querySelector('.bd-example').classList).filter(x => x !== 'bd-example').join(' ')
```

Comment:
```suggestion
      const classes = Array.from(event.target.closest('.bd-code-snippet').querySelector('.bd-example').classList).join(' ')
```

Suppose could live with a `bd-example` class inside body

Reply:
If you have a strong opinion on this I let you commit your suggestion. Otherwise I would prefer to remove everything that's not needed in order to not disturb the users.